### PR TITLE
Pretty Human Readable Output of Validate CLI

### PR DESCRIPTION
**Pull request by Thels**
Introducing a `-pretty` flag to the validate command.

image

Also introduced the ability to `watch` coupled with `tsup`. While ideally these wouldn't be coupled into one PR I made sure `watch` was working when developing this very PR.

### CODE REVIEW

**Review by willosborne — DISMISSED** [review]

File: shared/src/commands/validate/validate.ts [hunk 1]
```diff
@@ -42,38 +43,34 @@ export function exitBasedOffOfValidationOutcome(validationOutcome: ValidationOut
     process.exit(0);
 }
 
-export function getFormattedOutput(
+export enum OutputFormat {
```
Comment (aidanm3341) [inline shared/src/commands/validate/validate.ts:46]: enum's are generally avoided in Typescript - unless there's a reason I'm missing, I think we should do `type OutputFormat = 'junit' | 'json' | 'pretty'` instead. That also eliminates the need for a default branch in the switch statement below

File: shared/src/commands/validate/validate.ts [hunk 2]
```diff
@@ -295,7 +279,7 @@ export async function validate(
 
         await loadMetaSchemas(ajv, metaSchemaPath);
 
-        logger.info(`Loading pattern from : ${jsonSchemaLocation}`);
+        logger.info(`Loading MARKER pattern from : ${jsonSchemaLocation}`);
```
Comment (aidanm3341) [inline shared/src/commands/validate/validate.ts:282]: `MARKER`?

Comment (Thels) [reply shared/src/commands/validate/validate.ts:282]: Ah, yes - this was what I was adding/removing to determine if the multi-package `watch` was working with the custom `watch.mjs` script :) Will remove this prior to seeking final approval/merge.

Comment (Thels) [reply shared/src/commands/validate/validate.ts]: This is now removed - resolving.

**Review by aidanm3341 — APPROVED** [review]

**Pull request merged**